slasher/db/kv/spanner.go

```diff
@@ -72,9 +78,7 @@ func persistSpanMapsOnEviction(db *Store) func(key interface{}, value interface{
 }
 
 // Unmarshal a span map from an encoded, flattened array.
-func unmarshalSpan(ctx context.Context, enc []byte) (types.Span, error) {
```

Comment:
Removed the span here since this function is super short, span took more time than actual processing